src/averageAges.js

```diff
@@ -37,7 +46,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter(person => person.sex === 'f'
```

Comment:
You repeat this action in every function so create a separate function to get peope with sex that you need in different situations